continuous_delivery_scripts/assert_news.py

```diff
@@ -98,6 +98,12 @@ def validate_news_files(git: GitWrapper, root_dir: str, news_dir: str) -> None:
         validate_news_file(absolute_file_path)
 
 
+def convert_to_string_iter(list: Optional[list[Any]]) -> Iterable[str]:
```

Comment:
this is leading to the following flake8 issue `D103 Missing docstring in public function`
Could it be renamed so that this is no longer considered as an exposed function
```suggestion
def _convert_to_string_iter(list: Optional[list[Any]]) -> Iterable[str]:
```